media/plg_media-action_smartcrop/js/smartcrop.js

```diff
@@ -114,12 +114,35 @@ Joomla.MediaManager.Edit = Joomla.MediaManager.Edit || {};
 		}
 	}
 
+	// Add Delete Button
+	function addDeleteButton() {
+		var deleteDiv = document.createElement("div");
+		deleteDiv.className = "control-group";
+
+		var deleteButton = document.createElement("BUTTON");
+		deleteButton.setAttribute("type", "button");
+		deleteButton.setAttribute("id", "delete-focus");
+		deleteButton.setAttribute("class", "btn btn-danger");
+		deleteButton.appendChild(document.createTextNode("Delete Focus"));
+		deleteButton.addEventListener('click', function() {
+			Joomla.request({
+				url: resolveBaseUrl() + "/administrator/index.php?option=com_media&task=adaptiveimage.deleteFocus&path=" + path,
+				method: 'GET',
```

Comment:
can't we go for a `POST` here?

Reply:
```POST``` will work fine here, but I guess ```GET``` requests are faster.